### ISSUE TITLE
Proposal: About the date display of report.

### ISSUE THREAD

**Issue by 4f8p** [allinurl/goaccess#1310]
![date 1](https://user-images.githubusercontent.com/37243867/48970043-a74eb700-f04a-11e8-9443-a98acea49a58.png)

⚠️ **The image has been separated slash. But properly, use hyphen(-).**

Since it became i18n compatible, we suggest that the date format be also conformed to [ISO8601(RFC3339)](https://tools.ietf.org/html/rfc3339).

Example: `2018-01-01 -- 2018-11-25`

**Comment by allinurl** [comment]
I agree and like you said, i18n is now out, so it makes more sense have it as ISO8601.

**Comment by 4f8p** [comment]
![date](https://user-images.githubusercontent.com/37243867/49284606-69351580-f4d8-11e8-9165-e0e0d7ab0796.png)

It becomes unnatural if it corresponds to `ISO 8601` in the current design.
Therefore, we propose inserting character string before displaying the date.
We prepared two kinds of design plans above. Please consider.

**Comment by allinurl** [comment]
Agree on this change.
